mathics/core/parser/ast.py

```diff
@@ -105,10 +137,17 @@ def __repr__(self):
 
 
 class String(Atom):
+    """
+    A string is is pretty much the same as in any other programming language, a sequence of characters.
+    Having this in a class is useful so that we can distinguish it from Symbols.
+    The display of a String is surrounded by double quotes.
+    """
+
     def __repr__(self):
         return '"' + self.value + '"'
 
 
+# rocky: I really don't get why Filename has to be special, but so be it.
```

Comment:
I also don't get it.

Reply:
This is just because we choose different print representations: Strings are rounded by quotes, and filenames don't.

Reply:
Ok. I guess that makes sense, but then why isn't Filename distinct from Symbol? 

Is this how Mathematica does things too? I would have hoped they would say use another pair of delimiters like `<Filename>`.

I'll add a comment about this upon learning that `__repr__` really is the "right" thing to do.